# AUTOGENERATED! DO NOT EDIT! File to edit: dev/8.6_interpretability.prototypes_criticisms.ipynb (unless otherwise specified).

__all__ = ['load_np_vectors']

# Cell

import numpy as np
import pandas as pd

# ds4se
import ds4se
from ..clusterization import *

# Cell

def load_np_vectors(path: str) -> np.array:
    if not os.path.exists(path):
        logging.error("Vectors could not be found")
        return
    return np.load(path)